### PR TITLE
Sandbox documentation updates

### PR DESCRIPTION
**Pull request by nickrobison-usds**
**Why**

This makes some required updates to the documentation ahead of the sandbox release. Trying to follow along with the related confluence [doc](https://confluenceent.cms.gov/display/DAPC/Updates+to+screen+text+after+token+changes).

**What Changed**

Updated the attribution section in the documentation to be explicit about what constitutes a treatment related purpose. Will definitely need a wording review.

Fixed several little issues with the Swagger docs:
- Updated the scheme to use https (except for running locally) which avoids the redirect
- Removed the example from the `/metadata` endpoint, which was causing the browser to freeze due to the size of the json payload.
- Added explicit `@Path` annotations to all of our resources that mark them under the `/v1` root, not sure why this wasn't been propagated correctly, but it is now.
- Added missing Auth annotations to the `/Practitioner` resource (just the annotations were missing, not the actual auth!)

**Choices Made**

This change does not include the remapping of the MBIs, I wanted to get it out quickly for the wording review and for touching all the resources.

**Checklist**

- [x] All tests are passing via `make ci-app` (app change) and `make ci-web` (website change)
- [x] Swagger documentation has been updated
- [x] FHIR documentation has been updated
- [x] Any required dpc-ops changes have a PR submitted and mentioned in this ticket
- [x] Any manual migration steps are documented, scripts written (where applicable), and tested
- [x] Before merging, any required dpc-ops changes have been approved and merged into master of the dpc-ops repo

### CODE REVIEW

File: dpc-web/app/views/pages/reference.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ In addition, the only available resource types are those exposed by [Blue Button
 In order to receive data from the DPC application, a healthcare provider must have a treatment related purpose for viewing a patient's claims history.
 Providers can attest to their treatment purposes by submitting a an *attribution roster* which lists the patients currently under their care.
 
+In order for a provider to establish a treatment related purpose for viewing patient data, they must fulfill one of the following conditions:
```
Comment (nickrobison-usds) [inline dpc-web/app/views/pages/reference.md]: @amgleason Here's the first draft of the wording for the attribution requirements. Wordsmith away!

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/BaseResource.java [hunk 2]
```diff
@@ -67,6 +70,7 @@ public String version() {
     @Path("/metadata")
     @Public
     @ApiOperation(value = "Get FHIR Metadata", notes = "Returns the FHIR Capabilities statement for the application", response = CapabilityStatement.class)
+    @ApiResponses(@ApiResponse(code = 200, message = "successful operation", examples = @Example(@ExampleProperty(value = ""))))
```
Comment (embh) [inline dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/BaseResource.java]: Tiny change to be consistent with other response messages:
```suggestion
    @ApiResponses(@ApiResponse(code = 200, message = "Successful operation", examples = @Example(@ExampleProperty(value = ""))))
```

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/OrganizationResource.java [hunk 3]
```diff
@@ -19,6 +19,7 @@
 import java.util.stream.Collectors;
 
 @Api(value = "Organization")
```
Comment (embh) [inline dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/OrganizationResource.java]: Should the `@Api` annotation have `authorizations`?

File: dpc-web/app/views/pages/reference.md [hunk 4]
```diff
@@ -35,6 +35,13 @@ In addition, the only available resource types are those exposed by [Blue Button
...
+
+1. Have an existing treatment relationship, defined as a processed claim for the given patient with the provider's [National Provider Identity (NPI)](https://www.cms.gov/Regulations-and-Guidance/Administrative-Simplification/NationalProvIdentStand/) number within the past 18 months.
```
Comment (amgleason) [inline dpc-web/app/views/pages/reference.md]: Have an active treatment relationship, defined as a visit or a processed claim for the given patient with the provider's [National Provider Identity (NPI)](https://www.cms.gov/Regulations-and-Guidance/Administrative-Simplification/NationalProvIdentStand/) number within the past 18 months.

Comment (amgleason) [reply dpc-web/app/views/pages/reference.md]: With each roster addition or renewal, the provider is attesting that there is an active treatment relationship that creates a need for the data being requested.

File: dpc-web/app/views/pages/reference.md (hunk 1 above)
Comment (amgleason) [reply dpc-web/app/views/pages/reference.md]: @nickrobison-usds I commented below in the text

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/BaseResource.java (hunk 2 above)
Comment (nickrobison-usds) [reply dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/BaseResource.java]: Good catch!

**Pull request merged**